pkg/chaosdaemon/server.go

```diff
@@ -17,6 +17,8 @@ import (
 	"fmt"
 	"net"
 
+	"github.com/pingcap/chaos-mesh/pkg/utils"
```

Comment:
Is it better to group the import?

Reply:
Yes. Same issue as @mapleFU . Will fix it.